tasks/deploy.yml

```
@@ -32,4 +33,6 @@ tasks:
         git add -A
         git commit -m "{{.message}}" --allow-empty
         git push origin {{.branch}}
-
+        if [ "" == "{{.push}}" -o "true" == "{{.push}}" ]; then
```

```suggestion
        if [ "{{.push}}" == "" ] || [ "{{.push}}" == "true" ]; then
```
just for consistency with the rest of the codebase - but TIL `-o`!

---

I think if you use `||` you need to use `[[`, which has different semantics.

https://linuxhint.com/bash_operator_examples/#o51
https://linuxhint.com/bash_operator_examples/#o17

and https://linuxhint.com/bash_operator_examples/#o17